pandas/core/generic.py

```diff
@@ -2967,11 +2967,33 @@ def add_prefix(self, prefix):
 
         Parameters
         ----------
-        prefix : string
+        prefix : str
+            The string to add before each item name.
 
         Returns
         -------
-        with_prefix : type of caller
+        Series
+            Original Series with updated item names.
+
+        See Also
+        --------
+        pandas.Series.add_suffix: Add a suffix string to panel items names.
```

Comment:
I think the preferred option is to not user the `pandas.` prefix.

Reply:
I took it away. :)